packages/app/cypress/e2e/specs_list_actual_git_repo.cy.ts

```diff
@@ -89,7 +89,7 @@ describe('Spec List - Last updated with git info', () => {
     cy.get('[data-cy-row="foo.spec.js"] [data-cy="git-info-row"] svg')
     .trigger('mouseenter')
 
-    cy.get('.v-popper__popper--shown').contains('Created')
+    cy.get('.v-popper__popper--shown').should('be.visible').and('contain.text', 'Created')
```

Comment:
I wonder if we should add a custom command for this, like `cy.assertPopper` 🤔

Reply:
Are you thinking something like `cy.get('my-thing').assertPopper('be.visible')` to validate 'my-thing' contains a visible popper element? Or something more like `cy.get('my-thing').findPopperContent().should('be.visible')`? The approach might be impacted by the way the poppers are structured - the trigger is on the thing you're looking at, but the actual popper content is appended to `body` so a normal "find within" context wouldn't make sense

Reply:
I didn't have a specific implementation in mind, just saw the duplication and thought a command might be nice. Off the top of my head, maybe something simple:

```js
Cypress.Commands.add('popperContains', (text) => {
  return cy.get('.v-popper__popper--shown').should('be.visible').and('contain.text', text)
})
```

You can dynamically register it for this spec only, it wouldn't need to be globally available. I don't think it's a requirement for this PR, but I wonder if we should explore more of these utility commands for single spec files that contain a lot of duplication.